simple/src/main/java/org/apache/commons/rdf/simple/experimental/AbstractRDFParser.java

```diff
@@ -240,7 +240,7 @@ public T rdfTermFactory(RDF rdfTermFactory) {
     public T contentType(RDFSyntax rdfSyntax) throws IllegalArgumentException {
         AbstractRDFParser<T> c = clone();
         c.contentTypeSyntax = Optional.ofNullable(rdfSyntax);
-        c.contentType = c.contentTypeSyntax.map(syntax -> syntax.mediaType);
+        c.contentType = c.contentTypeSyntax.map(syntax -> syntax.mediaType());
         return c.asT();
```

Comment:
It's just a generics trick so that subclasses of `AbstractRDFParser` can return their own type in all the setter methods rather than `RDFParser` or `AbstractRDFParser`, which would hide any additional methods they specialize with.